dashboard/app/models/course.rb

```diff
@@ -17,8 +17,9 @@
 class Course < ApplicationRecord
   # Some Courses will have an associated Plc::Course, most will not
   has_one :plc_course, class_name: 'Plc::Course'
-  has_many :course_scripts, -> {order('position ASC')}
+  has_many :course_scripts, -> {where(experiment_name: nil).order('position ASC')}
   has_many :scripts, through: :course_scripts
+  has_many :alternate_course_scripts, -> {where.not(experiment_name: nil)}, class_name: 'CourseScript'
```

Comment:
Rather than eliminating these accessor as we talked about earlier, I've simply excluded alternate scripts from being retrieved by `course_scripts` and `scripts`. If it would be more clear I could rename these to `default_course_scripts` and `default_scripts`.

Reply:
Yeah, I'd prefer `default_course_scripts`.

Most of the bugs I ran into happened because the default accessor returns a reasonable default and is therefore *usually* correct, so it's easy for people to forget to handle the case of having multiple possible variants. I'd worry that somebody would come along and use `Course::course_scripts` without considering experimental swaps, breaking something for people in an experiment.

I guess that's still possible with `default_course_scripts`, but probably less likely.